zmon_aws_agent/main.py

```diff
@@ -147,6 +148,7 @@ def main():
         dynamodbs = aws.get_dynamodb_tables(region, infrastructure_account)
         certificates = aws.get_certificates(region, infrastructure_account)
         aws_limits = aws.get_limits(region, infrastructure_account, apps, elbs)
+        sqs = aws.get_sqs_queues(region, infrastructure_account)
```

Comment:
I think we applied some "tricks" like applying a modulo calculation to decide when to trigger some polling at different intervals. Long term, I agree that it should be like you suggested. thoughts @mohabusama ?